lua/mason-nvim-dap/mappings/adapters.lua

```diff
@@ -0,0 +1,57 @@
+local settings = require('mason-nvim-dap.settings')
+
+local M = {}
+
+-- https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#go-using-delve-directly
+M.delve = {
+	type = 'server',
+	port = '${port}',
+	executable = {
+		command = 'dlv',
+		args = { 'dap', '-l', '127.0.0.1:${port}' },
+	},
+}
+
+M.python = {
+	type = 'executable',
+	command = vim.fn.exepath('python3'),
+	args = { '-m', 'debugpy.adapter' },
```

Comment:
Shouldn't this be: `command = "debugpy-adapter"` since Mason installs the debugger and sets it up in the path at this command. with no args (the command calls the module correctly)

Reply:
I have also tested this with virtual environments and it all works out perfectly with this command even if I am editing code with a different environment activated

Reply:
This is magic. Thank you for this!